packages/llamaindex/src/vector-store/types.ts

```diff
@@ -18,6 +18,9 @@ export enum VectorStoreQueryMode {
   LINEAR_REGRESSION = "linear_regression",
   // maximum marginal relevance
   MMR = "mmr",
+
+  // for Azure AI Search
+  SEMANTIC_HYBRID = "semantic_hybrid",
```

Comment:
When accessing `VectorStoreQueryMode.SEMANTIC_HYBRID` from userland, we get the following error:

```
Cannot read properties of undefined (reading 'SEMANTIC_HYBRID')
```

@himself65 is there a better way to add a new query mode?

Reply:
Alright, then maybe that's just my local config. Thank you.